### DISCUSSION TITLE
Welcome to Rare Discussions!

### DISCUSSION THREAD

**Discussion by Dummerle** [RareDevs/Rare#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gnanini** [comment]
Hello Dummerle and Rare Community,

I'm Guilherme Nanini, I'm a beginner python coder and I want to contribute to this project (and learn)!

I'm tired of abstract exercises, I want to participate in a real life project!

Guilherme!

**Reply by Dummerle** [reply]
Thank you!
For version 1.0 I just want to make some bug fixes. In projects you can see some Features I want to add for version 2.0. Maybe you can implement one of these

**Reply by gnanini** [reply]
I did a module that catches the game steam code and return the game concept on protondb!

do you want me to figure out a way to get the codes?

**Reply by Dummerle** [reply]
That would be nice!

**Reply by gnanini** [reply]
i did it, it need some refinements, but it´s working!

If you input the correct game name, it outputs its id, then the other function gets a id and outputs its rank on protondb! It needs to generate a file containing a dic with games names as index and the games ids, it also may have a date index, so it can update the list each week...

now i´m figuring out how link them to the project

**Reply by gnanini** [reply]
the codes are on my repos, get_id and get_grade!

**Reply by Dummerle** [reply]
What do you mean with correct game name? Is it the app name or the title?

**Reply by Dummerle** [reply]
If it works, you can add these functions in the project at `rare/utils/utils.py`. I will add a note in Game info

**Reply by gnanini** [reply]
ok..

I'm presuming the games have the same name on epic and steam, I didn't developed a complex search algorithm, the game names are index on a dict and the function just calls them, otherwise it will be a problem...

**Reply by gnanini** [reply]
I already made some changes on the code, so now it records the json on a file, so it will not be requesting a full game list from steam api and it updates the json after 7 days!
